docs/src/main/paradox/scala/http/common/unmarshalling.md

```diff
@@ -51,14 +65,20 @@ The implicits for most of the predefined unmarshallers in Akka HTTP are provided
 `Unmarshaller` trait. This means that they are always available and never need to be explicitly imported.
 Additionally, you can simply "override" them by bringing your own custom version into local scope.
 
+@@@
+
 ## Custom Unmarshallers
 
 Akka HTTP gives you a few convenience tools for constructing unmarshallers for your own types.
-Usually you won't have to "manually" implement the `Unmarshaller` trait directly.
-Rather, it should be possible to use one of the convenience construction helpers defined on the `Unmarshaller`
-companion:
+Usually you won't have to "manually" implement the `Unmarshaller` @scala[trait]@java[class] directly.
+Rather, it should be possible to use one of the convenience construction helpers defined on
+@scala[the `Unmarshaller` companion]@java[`Unmarshaller`]:
+
+Scala
```

Comment:
For some reason the tab is not shown here when on the Scala page. On the Java page it is shown. Is that a known issue?

![screenshot from 2017-08-31 14 09 38](https://user-images.githubusercontent.com/9868/29922475-15286378-8e56-11e7-8754-7ddeb9e7f82b.png)

Reply:
This is Scala-only content since it talks about type aliases:

```
@@@ div { .group-scala }
Akka HTTP also predefines a number of helpful aliases for the types of unmarshallers that you'll likely work with most:

@@snip [package.scala]($akka-http$/akka-http/src/main/scala/akka/http/scaladsl/unmarshalling/package.scala) { #unmarshaller-aliases }

@@@
```

When clicking a tab, the page is reload and the current scroll state is lost. This might have been what happened.

Reply:
Just checked, indeed it shows this snippet (without tab) in the Scala version and it's completely hidden in the Java version, so that looks OK.